lgc/patch/FragColorExport.cpp

```diff
@@ -1005,3 +1029,33 @@ void FragColorExport::generateNullFragmentShaderBody(llvm::Function *entryPoint)
   BuilderBase builder(block);
   builder.CreateRetVoid();
 }
+
+// =====================================================================================================================
+// Copy MRT0's alpha to depth export alpha if enabled MRTZ.
+//
+// @param [in/out] info : The color export information for each color export in no particular order. Update the
+// channelMask if the MRT0's alpha is copied to MTRZ' alpha.
+// @param [in/out] values : The values that are to be exported. Indexed by the hw color target. Update the export value
+// for MTRZ
+// @param builder : The builder object that will be used to create new instructions.
+void FragColorExport::updateDepthExportAlpha(SmallVectorImpl<lgc::ColorExportInfo> &info,
+                                             SmallVectorImpl<Value *> &values, BuilderBase &builder) {
+  bool hasDepthExp = false;
+  bool hasMrt0 = false;
+  unsigned depthId = 0;
+  for (auto [id, exp] : enumerate(info)) {
+    if (exp.hwColorTarget == EXP_TARGET_Z) {
+      hasDepthExp = true;
+      depthId = id;
+    } else if (exp.hwColorTarget == EXP_TARGET_MRT_0) {
+      hasMrt0 = true;
+    }
+  }
+  if (hasMrt0 && hasDepthExp) {
+    auto mrt0Alpha = builder.CreateExtractElement(values[EXP_TARGET_MRT_0], 3);
+    auto &depthVal = values[EXP_TARGET_Z];
+    depthVal = builder.CreateInsertElement(depthVal, mrt0Alpha, 3);
+    // Update the mask with alpha channel enabled
+    info[depthId].location |= 8;
+  }
+}
```

Comment:
Is this still needed, given the change in `generateExportInstructions`?

Reply:
Removed.